src/Tizen.NUI/src/public/Common/StyleManager.cs

```diff
@@ -106,6 +106,29 @@ public static StyleManager Get()
             return ret;
         }
 
+        /// <summary>
+        /// Sets the brokenImageUrl
+        /// </summary>
+        /// <returns>A handle to the StyleManager control.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetBrokenImageUrl(uint type, string url)
```

Comment:
could you add parameter description?
/// <param name="type"> ... </param>
/// <param name="url"> ... </param>